README.md

```diff
@@ -10,13 +10,13 @@
 
 
 ## Installing the Canine CLI
-To install `canined` on your Linux machine, please go to [Releases](https://github.com/JACKAL-DAO/canine-chain/releases) and download the latest release. Move the executable to a folder in your `$PATH` and download [this](https://github.com/CosmWasm/wasmvm/raw/v1.1.1/internal/api/libwasmvm.x86_64.so) to `/lib/libwasmvm.x86_64.so` 
+To install `canined` on your Linux machine:
 
-```sh
-sudo wget https://github.com/CosmWasm/wasmvm/raw/v1.1.1/internal/api/libwasmvm.x86_64.so -O /lib/libwasmvm.x86_64.so
-```
+go install ./...
```

Comment:
Why not "make install"? It also injects the versions and stuff like that

Reply:
This route is simpler and there are other ways to inject versions.  I have seen in several occasions devs including me making errors happen by failing to update makefile.  No makefile no problem.

Reply:
This way doesn't assume any versions injecting though. I'd rather vote for a Makefile, but a proper one. Anyways it's not a blocker but something that should be considered for the sake of the easy installation.